src/styles/blocks/page.scss

```diff
@@ -0,0 +1,38 @@
+.page {
+  font-family: 'Space Grotesk', sans-serif;
+  font-weight: 700;
+  color: $main-text-color;
+  scroll-behavior: smooth;
+
+  &:has(.page__menu:target) {
```

Comment:
The :has() pseudo-class is not supported by any browsers yet. Consider using JavaScript or jQuery to achieve the desired effect.